cmake/testing.cmake

```diff
@@ -320,6 +320,9 @@ f3d_test(NAME TestInteractionDumpSceneState DATA dragon.vtu NO_BASELINE INTERACT
 f3d_test(NAME TestInteractionCycleVerbose DATA dragon.vtu ARGS --verbose -s NO_BASELINE INTERACTION REGEXP "Not coloring")#SSSSYC
 f3d_test(NAME TestInteractionEmptyDrop INTERACTION REGEXP "Drop event without any provided files.")#DropEvent Empty;
 f3d_test(NAME TestInteractionCameraUpdate DATA dragon.vtu INTERACTION DEFAULT_LIGHTS) #MouseWheel;MouseWheel;MouseWheel;S
+f3d_test(NAME TestInteractionFocalPointPicking DATA dragon.vtu INTERACTION DEFAULT_LIGHTS)
+f3d_test(NAME TestInteractionFocalPointPicking2 DATA dragon.vtu INTERACTION DEFAULT_LIGHTS)
+f3d_test(NAME TestInteractionFocalPointPicking3 DATA pointsCloud.vtp INTERACTION DEFAULT_LIGHTS)
```

Comment:
Lets try avoiding numbering of test but instead choose a descriptive name

Reply:
(the reason for this is that VTK support multiple baselines for a single test by numbering them. F3D does nt supports that and never will but still, lets make sure not to confuse anyone)